Add March 2023 newsletter (#1907)
writethedocs/www: docs/blog/newsletter-march-2023.rst

.. post:: March 06, 2023
   :tags: newsletter

#########################################
Write the Docs Newsletter – March 2023
#########################################

Hello, everyone! March is here, which should mean the changing of the season. I hope you are able to find some balance with the upcoming equinox.

This month's big community news is the `announcement of the Portland conference speakers <https://www.writethedocs.org/conf/portland/2023/news/announcing-speakers/>`__. This marks the return of in-person talks, and the variety of topics and speakers means it's sure to be a treat for those able to attend. If you can't make it to Portland, some other areas are also retuning to in-person meetups. So if that's your thing, check out the list at the end of the newsletter.

> In between, enjoy some discussions about how to deal with being the sole documentarian where you are, how to break up your docs-as-code, what the most interesting projects some of us have worked on, and how to face the challenge of documenting AI (since we know from last month that it won't yet replace us). 

Reviewer: "In between" what? :-p

Author: ```suggestion
Before the meetup list, enjoy some discussions about how to deal with being a sole documentarian, how to break up your docs-as-code, what the most interesting projects some of us have worked on, and how to face the challenge of documenting AI (since we know from last month that it won't yet replace us). 
```

> In between, enjoy some discussions about how to deal with being the sole documentarian where you are, how to break up your docs-as-code, what the most interesting projects some of us have worked on, and how to face the challenge of documenting AI (since we know from last month that it won't yet replace us). 

Reviewer: ```suggestion
In between, enjoy some discussions about how to deal with being a sole documentarian, how to break up your docs-as-code, what the most interesting projects some of us have worked on, and how to face the challenge of documenting AI (since we know from last month that it won't yet replace us). 
```

-----------------------------------------
Loneliness of the "Lone-ly" Documentarian
-----------------------------------------

“Sometimes being surrounded by everyone is the loneliest because you'll realize you have no one to turn to.” — Soraya

> A recent discussion in `#lone-writer <https://writethedocs.slack.com/archives/CAL5Y0NB1>`__ elicited some responses that may relate to all documentarians. The essence of the posted question also appears in #career-advice: How to develop skills to feel successful throughout a career?

Reviewer: "that may relate to all documentarians" feels a little weak to me? ymmv

> A recent discussion in `#lone-writer <https://writethedocs.slack.com/archives/CAL5Y0NB1>`__ elicited some responses that may relate to all documentarians. The essence of the posted question also appears in #career-advice: How to develop skills to feel successful throughout a career?

Author: ```suggestion
A recent discussion in `#lone-writer <https://writethedocs.slack.com/archives/CAL5Y0NB1>`__ elicited some responses with value for all documentarians. The essence of the posted question also appears in #career-advice: How to develop skills to feel successful throughout a career?
```

Others mentioned having a localization voiceover gig to demonstrate using a chainsaw, working on contracts for a company that built and sold childcare facilities, writing instructions for a reading tracking software company, authoring an operations manual for a robotic systems defense contractor, and creating content for a bespoke application for the oil and gas industry. From the variety of examples, it can be said that documentarians are tasked with complex and exciting projects, even if it's just helping a parent write and edit books on sleight-of-hand magic.

The most interesting projects that documentarians have worked on showcase the diverse range of topics and industries they can adapt to and the complexity of the information they need to communicate. As technology continues to evolve and shape our world, documentarians will remain a crucial part of the process, ensuring that information is shared effectively and efficiently with a broad audience. It's clear that it's already a fascinating and challenging field, and it’s expected that more innovative and interesting projects will emerge in the future.

---------------------
AI for documentarians

> ---------------------

Reviewer: It's super not clear to me whether we're talking about docs for AI or AI for docs.

> ---------------------
> AI for documentarians
> ---------------------

Author: ```suggestion
----------------
Documenting AI
----------------
```

First up, if you're working on a machine learning (ML) tool, keep an eye on data lineage. It's important to distinguish between the known input for the ML tool and These analysis or predictions that the tool outputs. For example, this might mean labelling a prediction on a person's age as "Age analysis result" rather than "Age of subject". If you're interested, the following article present more on the subject: `Always Leave An Analysis Paper Trail <https://counting.substack.com/p/data-science-practice-101-always-leave-an-analysis-paper-trail-cc17079fae5a>`__, and `An Introduction to the Data Biography <https://weallcount.com/2019/01/21/an-introduction-to-the-data-biography/>`__.

A related challenge can be explaining to end users what AI is actually doing. One discussion participant had been interested in the `End-User-Centered Explainable AI Framework <https://weina.me/euca/>`__, a prototyping tool for explaining AI to users; another linked to an example of a documentation standard trying to be transparent about AI, IBM's `AI FactSheets 360 <https://aifs360.mybluemix.net/introduction>`__.

The community was also interested in the ethics of AI. They recommended a few pieces:

> - A paper called “On the Dangers of Stochastic Parrots”.

Reviewer 2: Is the link missing for On the Dangers of Stochastic Parrots? Or maybe there is a reason not to include the link. I just noticed and thought I'd mention it.

Author: ```suggestion
- A paper called `"On the Dangers of Stochastic Parrots" <https://dl.acm.org/doi/10.1145/3442188.3445922>`__.
```
